web/src/helpers/gfs.js

```diff
@@ -1,40 +1,40 @@
-import moment from 'moment';
+import { addHours, formatISO, getHours, startOfHour, subHours } from 'date-fns';
 
 import { protectedJsonRequest } from './api';
 
 const GFS_STEP_ORIGIN = 6; // hours
 const GFS_STEP_HORIZON = 1; // hours
 
 export function getGfsTargetTimeBefore(datetime) {
-  let horizon = moment(datetime).utc().startOf('hour');
-  while (horizon.hour() % GFS_STEP_HORIZON !== 0) {
-    horizon = horizon.subtract(1, 'hour');
+  let horizon = startOfHour(datetime);
```

Comment:
I think we need to add
``` let horizon = startOfHour(datetime ? datetime : new Date());```
here

Reply:
Wouldn't `let horizon = startOfHour(datetime ?? new Date())` make more sense if that's the case?
Assuming datetime is either undefined or null and not explicitly false.

Reply:
yeah sounds good. I think this could have been something to do with the tests not being happy for the layers.

Reply:
I'll add the nullish coalescing and then add back the tests and see if they work now.